README.md

```diff
@@ -398,6 +399,56 @@ it('should cycle elements in document tab order', () => {
 })
 ```
 
+### `async hover(element)`
+
```

Comment:
Let's try and make these examples more real-world like. I don't think they need to actually run with real implantation of components, but if they could just demonstrate the expected type of component to be tested with this that would be great.

May also help to note that this will not trigger pseudo states for css (I'm not sure whether that's possible).